t/02_interface.t

```diff
@@ -9,7 +9,7 @@ use Kubectl::CLIWrapper;
 {
   my $control = Kubectl::CLIWrapper->new(namespace => 'ns1');
   my $command = join ' ', $control->command_for('create', 'pod');
-  diag $command;
+  note $command;
```

Comment:
Thanks for making me notice "note". I only knew "diag" 👍

Reply:
Cool :)